src/widgets/CodeTooltip/styles.css

```diff
@@ -10,4 +10,5 @@
 .popover {
   max-width: 90vw;
   position: absolute;
+  z-index: 1500;
```

Comment:
Do you know if we still need `position: absolute` now that we are using z-index?

Reply:
We still need it. 
z-index without `position: absolute`  displays the popup at the bottom of the page instead of the desired position

Reply:
Got it, thanks!